jfrog-cli/utils/config/config.go

```diff
@@ -239,7 +241,14 @@ func convertIfNecessary(content []byte) ([]byte, error) {
 }
 
 func GetJfrogHomeDir() (string, error) {
-	if os.Getenv(JfrogHomeEnv) != "" {
+	// If JfrogHomeDirEnv exists, use it.
+	// Else if JfrogHomeEnv exists, use it by adding .jfrog at the end.
+	// The second variable was left only to support users relying on it before introducing JfrogHomeDirEnv,
+	// shouldn't be used otherwise.
+
```

Comment:
I suggest simplifying the above description comment and replace it with:
// The JfrogHomeEnv environment variable has been deprecated and replaced with JfrogHomeDirEnv 

I also suggest renaming the *userDir* local variable in this function to *userHomeDie*.